src/test/java/codeu/controller/ProfileServletTest.java

```diff
@@ -13,6 +13,7 @@
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
```

Comment:
Sorry, I still don't completely understand what a session is so can you please explain how you're testing this.